### DISCUSSION TITLE
Enable support on s390x for KEDA

### DISCUSSION THREAD

**Discussion by dilipgb** [kedacore/keda#5584] [Q&A / Need Help] [answered]
Hi, we are from IBM Z team working on enabling knative on s390x architecture. Since Knative started using KEDA as the event based autoscaler for kafka we are looking to enable support for KEDA. I have successful in building images for s390x locally and patched with knative. Our tests are successful but looking for images to be available to end users via official releases. I'm raising this question to understand what are community thoughts and any guidelines from community about enablement?

**Comment by zroubalik** [comment, marked as answer]
Discussed here: https://github.com/kedacore/keda/issues/6543
